src/System.Drawing.Common/tests/Helpers.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Reflection;
+
+namespace System.Drawing.Tests
+{
+    public static class Helpers
+    {
+        public static string GetEmbeddedResourcePath(string name)
```

Comment:
The name of this function is a bit misleading, I think. Perhaps "GetTestBitmapPath()" would be better.